From: somestupidgirl <sunnevanattsol@gmail.com>
Subject: [PATCH] lsof: drop the channel and nexus sources the drop lacks

The darwin libproc dialect's Makefile and Mksrc list dchannel.c and
dnexus.c -- skywalk channel and nexus reporting -- but lsof-76 ships
neither file, and nothing in the dialect calls into them.  make stops on
"No rule to make target dchannel.c".  They come out of both lists.

--- a/lsof/dialects/darwin/libproc/Makefile	2026-09-15 00:01:22
+++ b/lsof/dialects/darwin/libproc/Makefile	2026-09-15 00:01:22
@@ -29,11 +29,11 @@
 
 SRC=    ddev.c dfile.c dmnt.c dproc.c dsock.c dstore.c \
 	arg.c main.c misc.c node.c print.c proc.c store.c usage.c util.c \
-	dnetpolicy.c dchannel.c dnexus.c
+	dnetpolicy.c
 
 OBJ=	ddev.o dfile.o dmnt.o dproc.o dsock.o dstore.o \
 	arg.o main.o misc.o node.o print.o proc.o store.o usage.o util.o \
-	dnetpolicy.o dchannel.o dnexus.o
+	dnetpolicy.o
 
 MAN=	lsof.8
 
--- a/lsof/dialects/darwin/libproc/Mksrc	2026-09-15 00:01:22
+++ b/lsof/dialects/darwin/libproc/Mksrc	2026-09-15 00:01:22
@@ -14,7 +14,7 @@
 
 
 D=dialects/darwin/libproc
-L="dlsof.h ddev.c dfile.c dmnt.c dproc.c dproto.h dsock.c dstore.c dnetpolicy.c dchannel.c dnexus.c machine.h"
+L="dlsof.h ddev.c dfile.c dmnt.c dproc.c dproto.h dsock.c dstore.c dnetpolicy.c machine.h"
 
 for i in $L
 do
